### DISCUSSION TITLE
Problems when interacting with Server API Endpoints

### DISCUSSION THREAD

**Discussion by TarasRashkevych99** [datalust/seq-tickets#2220] [Help] [answered]
Hello, I am trying to interact with endpoints such as "api/events", but every time I get {"Error": "Not found."} as a response.
Could you please help me solve this problem?
P.S. I use a docker container to interact with the seq server

**Comment by KodrAus** [comment, marked as answer]
Hi @TarasRashkevych99 :wave: What do you get if you request just `/api`? That should include a JSON payload that starts like `{"Product":"Seq ♦ Machine data, for humans."...`.

If you don't see it, it's possible you're actually hitting Seq via it's limited ingestion port instead of its regular API port. The Docker container serves the API on port 80 in the container, and ingestion on port 5341. Usually, you'll map these onto the host in your `docker run` command with an argument like `-p 5341:80` so that you access Seq via its API port on the host's port 5341.

**Reply by TarasRashkevych99** [reply]
Thank you a lot! I got a little bit confused at first because if you follow the seq documentation for the setup with a docker container ([Docker setup](https://docs.datalust.co/docs/getting-started-with-docker#running-seq-in-a-docker-container)) the host port 5431 is bind with the ingestion port 5431 of the container and the API doc says that you should use port 5431 ([API doc](https://docs.datalust.co/docs/using-the-http-api)). Maybe it would be better to state more clearly in the doc that port 5431 is sometimes used as API port.

**Reply by KodrAus** [reply]
Ah that's a good point. We usually talk about `5341` in the docs to mean Seq's API, but suggesting you map the ingestion port to `5341` in the Docker setup makes it confusing. We'll fix that up :+1:
